change-beta/@azure-communication-react-d99780b3-de43-4231-b959-61152ac12f71.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "none",
+  "area": "improvement",
+  "workstream": "RTE",
+  "comment": "Add internal storybook page and a feature flag for RTE feature",
```

Comment:
For changelog, should we spell out Rich Text Editor instead of using the abbr.?

Reply:
It should be okay as long as we use the same workstream value (it's just needed to group everything for the release driver)

Reply:
I was thinking about the comment "Add internal storybook page and a feature flag for RTE feature".
However, you have a good point that these change log for the in progress features will not be included in the release changelog. So we're all good here

Reply:
I see, sorry misunderstood your comment because it was shown under the "workstream" previously